libraries/ts-command-line/tsconfig.json

```diff
@@ -11,7 +11,7 @@
       "node"
     ],
     "lib": [
-      "es5",
+      "es6",
```

Comment:
> "es6", [](start = 5, length = 7)

@iclanton does this make sense to you?  We're not doing that in any other projects

Reply:
If we're all-in on ES6, then we should update this categorically.  But I'd be careful about that and its implications (e.g. many developers are still on Node 6)

---
In reply to: [168650795](https://github.com/Microsoft/web-build-tools/pull/530#discussion_r168650795) [](ancestors = 168650795)

Reply:
Using "es2015.promise" instead

---
In reply to: [168651134](https://github.com/Microsoft/web-build-tools/pull/530#discussion_r168651134) [](ancestors = 168651134,168650795)